LICENSE

```diff
@@ -0,0 +1,25 @@
+Redistribution and use in source and binary forms, with or without
```

Comment:
Is there a reason why you didn't include the top part of the license?

```suggestion
BSD 3-Clause License

Copyright (c) 2008-2013, Willow Garage, Inc.
                     2021, PickNik Robotics
All rights reserved.

Redistribution and use in source and binary forms, with or without
```

Reply:
There are so many authors here...